cmd/hmcollector/hmcollector.go

```diff
@@ -83,6 +83,14 @@ var (
 	logInsecFailover = flag.Bool("hmcollector_log_insecure_failover", true, "Log/don't log TLS insecure failovers.")
 	httpTimeout      = flag.Int("http_timeout", 10, "Timeout in seconds for HTTP operations.")
 
+	// vars for custom logging configuration
+	logModesEnv        = ""
+	logXnamesEnv       = ""
+	shouldLogErrors    = false
+	shouldLogForXnames = false
+	logXnames          = make(map[string]struct{})
+	logAllowedModes    = []string{"errors"}
```

Comment:
"errors" is the only allowed log mode, or did you intend to add more to it?

Reply:
I don't have others in mind, but thought maybe someday another would be added.
Alternately I could just remove the LOG_MODES option, and always log the error. The reason I didn't do this original is because I'm worried about causing an issue with there being too much logging.

Reply:
Sounds good to me!